File: S5_C#/ejercicio4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.RegularExpressions;
using System.Threading.Tasks;

namespace S5_C_
{
    internal class ejercicio4
    {
        static void Main (string[] args)
        {
            double Resultado=0, Base, Altura, radio;
            Console.WriteLine("------------------- SISTEMA DE CALCULO DE AREA ------------\n");

            Console.WriteLine("***** MENU DE OPCIONES ******\n");
            Console.WriteLine("[1]. CUADRADO ");
            Console.WriteLine("[2]. RECTANGULO ");
            Console.WriteLine("[3]. TRIANGULO ");
            Console.WriteLine("[4]. CIRCULO ");
            Console.WriteLine("*****************************\n");

            Console.Write("ingrese una opcion: ");
            int opc = int.Parse(Console.ReadLine());

            Console.Clear();
           
            switch (opc)
            {
                case 1:
                    double lado;
                    Console.WriteLine("Ingrese el lado del cuadrado: ");
                    lado = double.Parse(Console.ReadLine());
                    Resultado = (lado * lado);
                    break;
                case 2:
                    Console.WriteLine("Ingrese la altura: ");
                    Altura = double.Parse(Console.ReadLine());
                    Console.WriteLine("Ingrese la base: ");
                    Base = double.Parse(Console.ReadLine());

                    Resultado = Base * Altura;
                    break;
                case 3:
                    Console.WriteLine("Ingrese la altura: ");
                    Altura = double.Parse(Console.ReadLine());
                    Console.WriteLine("Ingrese la base: ");
                    Base = double.Parse(Console.ReadLine());

                    Resultado = (Altura * Base)/2;
                    break;
                case 4:
                    Console.WriteLine("Ingrese el radio: ");
                    radio = double.Parse(Console.ReadLine());
                    Resultado = Math.PI * Math.Pow(radio,2);
                    break;
                default: Console.WriteLine("ERROR DE TECLA"); break;

            }
            Console.WriteLine("El Area es: " + Resultado);
        }
    }
}

File: S5_PY/ejercicio4.py
from email.policy import default
import math
print("------------------- SISTEMA DE CALCULO DE AREA ------------\n")

print("***** MENU DE OPCIONES ******\n")
print("[1]. CUADRADO ")
print("[2]. RECTANGULO ")
print("[3]. TRIANGULO ")
print("[4]. CIRCULO ")
print("*****************************\n")

opc = int(input("ingresar opcion: "))

match opc:
    case 1:
        lado = float(input("Ingrese el lado: "))
        Area = float(lado * lado)
    case 2:
        base = float(input("Ingrese la base: "))
        Altura = float(input("Ingrese la altura: "))
        Area = float(Altura*base)
    case 3:
        base = float(input("Ingrese la base: "))
        Altura = float(input("Ingrese la altura: "))
        Area = float((Altura*base)/2)
    case 4:
        radio = float(input("Ingrese el radio: "))
        Area = math.pi * math.pow(radio,2)
    case _:
        print("Opcion invalida")
print("----- RESULTADO ---------")
print(f"El area es: {Area}")
print("-------------------------")